src/connectedmachine/azext_connectedmachine/vendored_sdks/connectedmachine/py.typed

```diff
@@ -0,0 +1 @@
+# Marker file for PEP 561.
```

Comment:
what's this file used for?

Reply:
No idea it was there as a part of the generation